keps/sig-node/5365-ImageVolume-with-image-digest/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: KEP Template
+kep-number: 5365
+authors:
+  - "@iholder101"
+owning-sig: sig-node
+participating-sigs:
+status: implementable #provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-06-05
+reviewers:
+  - TBD
+  - "@alice.doe"
+approvers:
+  - TBD
+  - "@oscar.doe"
```

Comment:
please fill up, you can put me or @haircommander s reviewers and I would suggest @mrunalp as approver as he is an approver for #4639

Reply:
Thanks very much to the both of you!
Added :+1: